examples/advanced/gif_animation/main.py

```diff
@@ -0,0 +1,99 @@
+import blenderproc as bproc
+import numpy as np
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('scene', nargs='?', default="examples/resources/scene.obj", help="Path to the scene.obj file")
+parser.add_argument('output_dir', nargs='?', default="examples/advanced/gif_animation/output", help="Path to where the final files, will be saved")
+args = parser.parse_args()
+
+bproc.init()
+
+# Set a global number of frames for your final animation
+frame_end = 99
+
+# load the objects into the scene
+objs = bproc.loader.load_obj(args.scene)
+
+# Set all entities from the scene as solid, passive objects
+for obj in objs:
+    obj.enable_rigidbody(active=False)
+
+# create an additional item in the scene that is moveable
+item = bproc.object.create_primitive("CONE")
+item.set_location([0,-1,7])
+item.set_rotation_euler([1,0,1])
+item.enable_rigidbody(active=True)
+
+# Aktivate the physics simulation
+bproc.object.simulate_physics(
+    min_simulation_time=4,
+    max_simulation_time=20,
+    check_object_interval=1
+)
+
+# Set how many frames your animation should contain
+bproc.utility.set_keyframe_render_interval(frame_end=frame_end)
+
+# define a light and set its location and energy level
+light1 = bproc.types.Light()
+light1.set_type("POINT")
+light1.set_location([5, -5, 5])
+light1.set_energy(1000)
+
+# add a second red blinking light source
+light2 = bproc.types.Light()
+light2.set_type("POINT")
+light2.set_color([1,0,0])
+light2.set_location([-5,-5,5])
+
+# Define the rythm of the blinking
+frame = 0
+on_frames = []
+while frame < frame_end:
+    on_frames.append(frame)
+    frame += 1
+    if frame % 10 == 0:
+        frame += 10
+
+for frame in range(frame_end):
+    if frame in on_frames:
+        light2.set_energy(600, frame=frame)
+    else:
+        light2.set_energy(0, frame=frame)
```

Comment:
It took me a moment to get what it is happening here, but you are trying to turn the light on for ten frames and then turn it off for ten frames?

```suggestion
for frame in range(frame_end):
    # this changes every ten frames from on to off and vice-versa
    if (frame // 10) % 2 == 0:
        light2.set_energy(600, frame=frame)
    else:
        light2.set_energy(0, frame=frame)
```